sources/packages/web/src/components/students/applicationTracker/ApplicationProgressBar.vue

```diff
@@ -123,15 +124,11 @@ export default defineComponent({
     const trackFillColor = computed<string>(() => {
       if (statusIconDetails.value.statusType === "error") {
         return "error";
-      }
-      if (
-        props.applicationStatus === ApplicationStatus.Completed &&
-        applicationProgressDetails.value.appealStatus !==
-          StudentAppealStatus.Pending
-      ) {
+      } else if (statusIconDetails.value.statusType === "success") {
```

Comment:
@dheepak-aot  @guru-aot do you why ` if (
        props.applicationStatus === ApplicationStatus.Completed &&
        applicationProgressDetails.value.appealStatus !==
          StudentAppealStatus.Pending
      )` was done instead of `(statusIconDetails.value.statusType === "success")` before, just wanted to check, if there was a reason

Reply:
I am not sure of the exact reason @ann-aot . When I had a look at the first PR of #1683 I see the status icon was missing on various scenarios on completed status. But the upcoming PRs added the status icon back on these scenarios I guess.  But not 100% sure.

###PR 1
![image](https://github.com/bcgov/SIMS/assets/54600590/d9ae8ba2-0135-494e-8ec7-b18bfa868a8e)